client/a8c-for-agencies/sections/sites/features/a4a/site-details.tsx

```diff
@@ -12,25 +15,29 @@ export default function SiteDetails( { site }: any ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	/* eslint-disable-next-line */
-	const { a4a_site_id: siteId, a4a_site_tags: initialTags } = site;
+	const { a4a_site_id: siteId, a4a_site_notes: initialNotes, a4a_site_tags: initialTags } = site;
 
 	const [ tags, setTags ] = useState(
 		initialTags ? initialTags.map( ( tag: SiteTagType ) => tag.label ) : []
 	);
+
+	const [ notes, setNotes ] = useState( initialNotes ?? [] );
+
 	const [ isLoading, setIsLoading ] = useState( false );
-	const { mutate } = useUpdateSiteTagsMutation();
+	const { mutate: mutateTags } = useUpdateSiteTagsMutation();
+	const { mutate: mutateNotes } = useCreateSiteNoteMutation();
 	const queryClient = useQueryClient();
 
-	const onAddTags = ( siteTags: string[] ) => {
+	const onCreateNote = ( note: string ) => {
 		setIsLoading( true );
```

Comment:
react-query already handles the state of queries - let's use that instead of tracking status on our own potentially introducing bugs along the way, e.g.:
`const isLoading = !tagsMutation.isIdle || !notesMutation.isIdle` (there may be better status props to use, it's been a while since I last read the react-query docs)